generate-training-data.py

```diff
@@ -11,3 +11,58 @@
 #           |-- paper\
 #           |-- scissors\
 # having a folder structure like this will make it easy for you to read the images while pre-processing
+
+
+
+def get_images():
+    
+    cap = cv2.VideoCapture(0)
+    cnt1=0
+    while(cnt1<=250):
+        ret , frame = cap.read()
+        if ret == True:
+            frame = cv2.rectangle(frame,(100,100),(300,300),(0,0,255),5)
+            cv2.imshow('frame',frame)
+            #for i in range(2011):
+            cv2.imwrite('img'+str(cnt1)+'.jpg',frame[100:300,100:300])
+            cnt1=cnt1+1
+                
+        if cv2.waitKey(1) & 0xff == ord('q'):
+            break
+        
+    cap.release()
+    cv2.destroyAllWindows()
+
+    for i in range(50):
+        os.remove('img'+str(i)+'.jpg')
```

Comment:
This is a nice touch, removing the first 50 images. 
This essentially gives the user some time to get ready.

You could have handled it a little better if this was your intension, maybe say like asking him to press a key if he is ready then starting to save images instead of starting to save images as soon as the video starts.

Just a thought for improvement.

Reply:
Yeah i thought of that too but found it more convenient. I will still try it out! Thanks!

Reply:
Actually i wanted to try it out on my own. So my team mate doesn't know about it. My team mate is Thejasvin BM. He will be making a separate pr.